siphon/simplewebservice/wyoming.py

```diff
@@ -11,8 +11,12 @@
 import numpy as np
 import pandas as pd
 
-from .._tools import get_wind_components
-from ..http_util import HTTPEndPoint
+import sys, os
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
I’m guessing this is working around problems trying to run siphon from git on your system? Did you properly install Siphon as described in the [Contributor’s Guide](https://github.com/Unidata/siphon/blob/master/CONTRIBUTING.md#pull-requests)?